packages/expect/__typetests__/expect.test.ts

```diff
@@ -5,12 +5,77 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {expectError} from 'tsd-lite';
-import type * as expect from 'expect';
+import {expectError, expectType} from 'tsd-lite';
+import type {EqualsFunction, Tester} from '@jest/expect-utils';
+import {type Matchers, expect} from 'expect';
+import type * as jestMatcherUtils from 'jest-matcher-utils';
 
-type M = expect.Matchers<void, unknown>;
-type N = expect.Matchers<void>;
+type M = Matchers<void, unknown>;
+type N = Matchers<void>;
 
 expectError(() => {
-  type E = expect.Matchers;
+  type E = Matchers;
 });
+
+// extend
+
+type MatcherUtils = typeof jestMatcherUtils & {
```

Comment:
we should export this type probably

Reply:
Hm.. But this is just a helper for testing. Perhaps that’s fine?

Reply:
it's a copy of https://github.com/facebook/jest/blob/60eb4160b5258eed5faf381e0c8ea6662076cc6c/packages/expect/src/types.ts#L46-L49, no?

Reply:
Got it. I though you speak about exporting `typeof jestMatcherUtil` somehow.

Reply:
Do you think `MatcherUtils` type should be public? Type test is importing from build, not from source.

Reply:
it was just some duplication that felt unnecessary, I don't feel strongly here 🙂